ontology/yaml/resources/HVAC/entity_types/RSR.yaml

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+RSR_HHCDM:
```

Comment:
nothing in this file, nor hwrsr or chwrsr, is used by anything. should they be removed?